### ISSUE TITLE
Introduce tests of fingerprint cosinstency

### ISSUE THREAD

**Issue by petrpatek** [apify/fingerprint-injector#18]

**Comment by JaneJeon** [comment]
+1 for this. `fingerprint-injector` + using actual Chrome is literally enough to avoid detection from basically all websites that I've tested on, _except_ for CreepJS.

It's actually kinda crazy, but there are only a few points that are detected by creepJS (mainly the navigator) - so I believe testing with this to try to advance the "arms race" would be absolutely necessary for this project's longevity.

**Comment by petrpatek** [comment]
Exactly, I am glad that it works for you! From my perspective, I want to start testing to validate if the improvements we are doing are improvements at all.

**Comment by JaneJeon** [comment]
Well, for what it's worth, CreepJS apparently detected "lies" in the navigator (both the main thread and the service worker) that the injector either didn't cover or they used some magic to detect tampering with it (still trying to figure out which is the case).

Obviously YMMV
